src/java/htsjdk/samtools/SAMFileWriterFactory.java

```diff
@@ -293,16 +291,39 @@ public SAMFileWriter makeSAMOrBAMWriter(final SAMFileHeader header, final boolea
         return makeBAMWriter(header, presorted, outputFile);
     }
 
+    /**
+     *
+     * Create a SAM, BAM or CRAM writer based on examination of the outputFile extension.
+     * For CRAM files, read names and tags are preserved.
```

Comment:
I didn't know this about cram format, and I suspect a lot of people will also not know it.  I think it's more confusing than helpful at the moment.  Possibly something more detailed would be helpful, but I'm having difficulty phrasing it.  

Maybe something like "CRAM compressed files will be written losslessly and maintain the same information as a BAM"